rivershared/startstop/main_test.go

```diff
@@ -0,0 +1,11 @@
+package startstop
+
+import (
+	"testing"
+
+	"github.com/riverqueue/river/rivershared/riversharedtest"
+)
+
+func TestMain(m *testing.M) {
+	riversharedtest.WrapTestMain(m)
+}
```

Comment:
Added this because I found some goroutines leaks in the tests. Fixed those, but this will prevent future ones from occurring.